### DISCUSSION TITLE
How to initiate lazyterm when starting nvim instance instead of lazy loading it

### DISCUSSION THREAD

**Discussion by idelice** [LazyVim/LazyVim#4501] [General]
This keymap

`map("n", "<c-/>", lazyterm, { desc = "Terminal (Root Dir)" })`

..is toggling lazyterm which is nice, but i always have to wait a second or two for it to boot up and i guess it's because it's lazy loaded (?)

Is it possible to load the terminal at the very first nvim instance so you can start typing your commands straight away instead of waiting few seconds for it to boot up?

**Comment by dpetka2001** [comment]
lazyterm is not lazy loaded. It's your shell configuration that has to be taken into consideration when the terminal boots up. I'm guessing you have some kind of addon for your shell that decorates your prompt. That's what is likely causing the delay you observe.

I'm also using fish shell with plugin `tide` to decorate my prompt and observe a 0.5-1 sec delay when lazyterm opens. But when I change in `options.lua` the following `LazyVim.terminal.setup("bash")` it opens instantly.

**Reply by idelice** [reply]
Thanks, I'll take a look at my shell configs to see if something is causing a delay.
